docs/References/Window.md

```diff
@@ -296,6 +296,9 @@ Turn the window's native shadow on/off. Useful for frameless, transparent window
 !!! warning "Behavior Changed"
     The behavior of the function is changed since 0.13.0. Please see [Migration Notes from 0.12 to 0.13](../For Users/Migration/From 0.12 to 0.13.md).
 
+!!! note
+    Since 0.75, the function throws an error if the build flavor is not SDK. This error can be caught in a try catch block which prevents the application from crashing.
```

Comment:
```suggestion
    Since 0.75, the function throws an error if the build flavor is not SDK. This error can be caught in a try catch block.
```
If the code just returns in Normal, then it won't crash at all.